src/themes/default/lv_theme_default.c

```diff
@@ -739,11 +739,11 @@ static void theme_apply(lv_theme_t * th, lv_obj_t * obj)
 #if LV_USE_TABVIEW
         lv_obj_t * parent = lv_obj_get_parent(obj);
         /*Tabview content area*/
-        if(parent && lv_obj_check_type(parent, &lv_tabview_class) && lv_obj_get_index(obj) == 1) {
+        if(parent && lv_obj_check_type(parent, &lv_tabview_class) && lv_obj_get_child(parent, 1) == obj) {
```

Comment:
```suggestion
        if(parent == NULL) return;

        if(lv_obj_check_type(parent, &lv_tabview_class) && lv_obj_get_child(parent, 1) == obj) {
```

And remove check to `parent` on line 753.

Reply:
Actually all `parent == NULL` check can be removed as it's already [checked](https://github.com/lvgl/lvgl/blob/master/src/themes/default/lv_theme_default.c#L731) at the beginning.